addons/website_slides/models/slide_slide.py

```diff
@@ -478,28 +478,28 @@ def _compute_embed_code(self):
                 if slide.video_source_type == 'youtube':
                     query_params = urls.url_parse(slide.video_url).query
                     query_params = query_params + '&theme=light' if query_params else 'theme=light'
-                    embed_code = Markup('<iframe src="//www.youtube-nocookie.com/embed/%s?%s" allowFullScreen="true" frameborder="0"></iframe>') % (slide.youtube_id, query_params)
+                    embed_code = Markup('<iframe src="//www.youtube-nocookie.com/embed/%s?%s" allowFullScreen="true" frameborder="0" aria-label="Youtube"></iframe>') % (slide.youtube_id, query_params)
```

Comment:
(Oops, missed that 👀 )

Can you change to YouTube with a capital "T"? They can be very picky about this.

In the same idea, I see you have a few "Linkedin", should be "LinkedIn" with a capital "I" I think.

Reply:
@awa-odoo , sure, I will change that, thanks for notifying! :pray: